core/src/main/java/io/seata/core/store/db/LogStoreSqls.java

```diff
@@ -113,9 +113,19 @@
     /**
      * The constant QUERY_GLOBAL_TRANSACTION_BY_STATUS.
      */
-    public static final String QUERY_GLOBAL_TRANSACTION_BY_STATUS = "select " + ALL_GLOBAL_COLUMNS + " from "
-        + GLOBAL_TABLE_PLACEHOLD +
-        " where " + ServerTableColumnsName.GLOBAL_TABLE_STATUS + " in (" + PRAMETER_PLACEHOLD + ") order by " + ServerTableColumnsName.GLOBAL_TABLE_GMT_MODIFIED + " limit ?";
+    public static final String QUERY_GLOBAL_TRANSACTION_BY_STATUS_MYSQL =
+            "select " + ALL_GLOBAL_COLUMNS + " from " + GLOBAL_TABLE_PLACEHOLD
+                    + " where " + ServerTableColumnsName.GLOBAL_TABLE_STATUS + " in (" + PRAMETER_PLACEHOLD + ")"
+                    + " order by " + ServerTableColumnsName.GLOBAL_TABLE_GMT_MODIFIED + " limit ?";
+
+    public static final String QUERY_GLOBAL_TRANSACTION_BY_STATUS_ORACLE =
+            "select * from ("
+                    + "  select tt.*, ROWNUM as rowno from ("
+                    + "    select " + ALL_GLOBAL_COLUMNS + " from " + GLOBAL_TABLE_PLACEHOLD
+                    + "    where " + ServerTableColumnsName.GLOBAL_TABLE_STATUS + " in (" + PRAMETER_PLACEHOLD + ")"
+                    + "    order by " + ServerTableColumnsName.GLOBAL_TABLE_GMT_MODIFIED + ") tt"
+                    + "  where ROWNUM <= ?) table_alias"
+                    + " where table_alias.rowno >= 0";
```

Comment:
Whether the outermost select can be removed

Reply:
i update the sql in my branch, i wonder how to synchronize to the pr?